libs/caseStatuses.js

```diff
@@ -62,6 +63,12 @@ const statuses = [
   },
   {
     type: ACTIVE_COMPLETION_REQUIRED_VIVA,
+    name: 'Ansökan behöver kompletteras',
+    description:
+      'Du har skickat in en ansökan för #MONTH_NAME. För att vi ska kunna behandla din ansökan finns det uppgifter som du behöver komplettera.\nKomplettering ska ha skickats in till oss senast #COMPLETION_DUEDATE.',
```

Comment:
For the newline to work you need to add two `\n` like `\n\n`. Not the best looking solution but it will work